resources/webroot/index.js.mustache

```diff
@@ -1842,21 +1842,37 @@ function addPokemon(pokemon, ts) {
   })
 
   if (oldPokemon === undefined || pokemon.is_start) {
+    var latOffset = Math.random() * 0.0003 - 0.0001
+    var lonOffset = Math.random() * 0.0003 - 0.0001
+
     if (pokemon.pokestop_id !== null && pokemon.spawn_id === null) {
```

Comment:
wouldn't ` pokemon.spawn_id === null` cover both cases here?
Is there a reason for duplicating the code here?